moderntribe/square-one
# Bug/links block rendering

## What does this do/fix?

1. The Links block renders nothing without list data. There is a content block above so this is confusing to the user as they add content and see nothing. All content added will now render.
2. The Link block uses nested repeaters that don't pass our null check in our Model->get method.  Why? ACF returns an array of data from repeaters, and the keys can have a null value for nested fields and in this case the links. This fixes the issue at the link controller level by checking for a null value and casting it back to a string.

## QA

Testing Instructions: 
1. Goto test X
2. VERIFY all content renders.
3. VERIFY when adding a new link to the repeater with no link added. Post Saved. No error.

Test Link:
- Pending deploy.

Links to relevant issues
- [Link to Issue](https://moderntribe.atlassian.net/browse/SQONE-629)
